idaes/core/components.py

```diff
@@ -37,7 +36,7 @@ class ComponentData(ProcessBlockData):
     CONFIG = ConfigBlock()
 
     CONFIG.declare("valid_phase_types", ConfigValue(
-            domain=list_of_phase_types,
+            domain=ListOf(PT),
```

Comment:
Super minor comment: PhaseType will be more descriptive as I was looking for what PT meant until I saw the imports. Feel free to ignore.